.changeset/gold-rocks-cry.md

```diff
@@ -0,0 +1,15 @@
+---
+'astro': minor
+'@astrojs/mdx': minor
+'@astrojs/markdown-remark': minor
+---
+
+Add a new experimental flag (`experimental.assets`) to enable our new core Assets story.
+
+This unlocks a few features:
+- Image component and JavaScript API to transform and optimize images.
+- Support for images stored relatively to your content in Markdown.
+- Support for validating assets using Content Collections.
+- and more!
```

Comment:
```suggestion
Add a new experimental flag (`experimental.assets`) to enable our new core Assets story.

This unlocks a few features:
- A new built-in image component and JavaScript API to transform and optimize images.
- Relative images with automatic optimization in Markdown and MDX files.
- Support for validating assets using content collections.
- and more!
```
Just suggesting some edits to make this more closely align with what we have on the assets page itself. 

I think emphasizing new and built-in for the image component could be helpful (since we're going to have to remind people they can't use the existing integration), and I think "relative images" in Markdown/MDX is the hot phrase everyone wants to hear!  🥳

Reply:
Just removed the `and MDX` part since this isn't actually supported at the moment, otherwise looks awesome! Thank you very much!